test/index.html

```diff
@@ -64,7 +64,7 @@
   </script>
 
   <!-- The script under test -->
-  <script src="../inert.js"></script>
+  <script src="../dist/inert.js"></script>
```

Comment:
Depends on the browsers we test I guess. If we want to verify things work in IE10 we'd have to always test the built version